components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/test/java/org/wso2/carbon/identity/mgt/endpoint/util/PreferenceRetrievalClientTest.java

```diff
@@ -53,6 +53,8 @@ public class PreferenceRetrievalClientTest {
     public static final String NOTIFICATION_PASSWORD_ENABLE_PROPERTY = "Recovery.Notification.Password.Enable";
     public static final String EMAIL_LINK_PASSWORD_RECOVERY_PROPERTY =
             "Recovery.Notification.Password.emailLink.Enable";
+    public static final String EMAIL_OTP_PASSWORD_RECOVERY_PROPERTY =
+            "Recovery.Notification.Password.OTP.SendOTPInEmail";
```

Comment:
Isn't it better to have the name Recovery.Notification.Password.EmailOTP.Enable?